### PR TITLE
[tools] Disallow unused variables

### PR DESCRIPTION
**Pull request by jugglinmike**
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/11097)

### CODE REVIEW

**Review by gsnedders — APPROVED** [review]

**Review by gsnedders — CHANGES_REQUESTED** [review]
per comment: flake8 config should be kept in sync across tools/tox.ini, tools/wpt/tox.ini, and tools/wptrunner/tox.ini

**Comment by jugglinmike** [comment]
You're right. Sorry I missed that

**Comment by jugglinmike** [comment]
@gsnedders I've updated the other two configuration files, removed the unused variables this exposed, and the tests have passed

**Review by foolip — APPROVED** [review]
Sweet!

File: tools/wptrunner/wptrunner/executors/base.py [hunk 1]
```diff
@@ -243,7 +243,7 @@ def logger(self):
         return self.executor.logger
 
     def get_hash(self, test, viewport_size, dpi):
-        timeout = test.timeout * self.timeout_multiplier
+        test.timeout * self.timeout_multiplier
```
Comment (gsnedders) [inline tools/wptrunner/wptrunner/executors/base.py]: This is dead code, just remove it.

Comment (jugglinmike) [reply tools/wptrunner/wptrunner/executors/base.py]: Good catch. I swear this was a function call last night.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**